### PR TITLE
feat: example gdml for pfrich/mrich review

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
This introduces an example gdml file included in a fake MRICH/PFRICH detector. All that should be needed on Monday is dropping in the correct GDML export (and likely shifting some positions).

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [x] Changes have been communicated to collaborators @osbornjd 

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No.

### Does this PR change default behavior?
No.

### CODE REVIEW

File: .github/workflows/linux-eic-shell.yml [hunk 1]
```diff
@@ -143,7 +144,7 @@ jobs:
         variables: |
           DETECTOR_REPOSITORYURL=${{ github.server_url }}/${{ github.repository }}
           DETECTOR_REPOSITORYREF=${{ github.ref }}
-          DETECTOR_CONFIG=${{ matrix.detector_config }}
+          DETECTOR_CONFIG=${{ github.repository }}_${{ matrix.detector_config }}
```
Comment (wdconinc) [inline .github/workflows/linux-eic-shell.yml:147]: Curious... GitHub has no context variable that would contain just `epic`...
```suggestion
          DETECTOR_CONFIG=epic_${{ matrix.detector_config }}
```

**Review by osbornjd — COMMENTED** [review]
[This](https://github.com/eic/epic/actions/runs/4263866465/jobs/7421293618) implies there is still an overlap, or is this an existing issue? Otherwise I will approve as I don't have any other comments

**Comment by wdconinc** [comment]
That's the geant4 bug that @johnlajoie and @ruse-traveler are working on narrowing down. The HCAL geometry itself is fine.

**Comment by osbornjd** [comment]
Okay, I didn't recall if that was still an open issue. Looks good to me then

**Review by osbornjd — APPROVED** [review]

**Pull request merged**